qiskit/pulse/commands/command.py

```diff
@@ -42,11 +45,27 @@ def __init__(self, duration: int = None, name: str = None):
         else:
             raise PulseError('Pulse duration should be integer.')
 
-        if name:
-            self._name = name
+        self._name = self._name = self.create_name(name=None, prefix='c', counter=self.instances_counter)
+
+    @staticmethod
+    def create_name(name: str = None, prefix: str = None, counter: itertools.count = None) -> str:
```

Comment:
I believe it might be more appropriate if this is a `classmethod`, in which case both the `prefix` and the `counter` can be called from the class variables.

Reply:
Agreed